frontend/src/component/pages/shared/PreferenceToggle.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { PreferenceToggle } from './PreferenceToggle';
+import { fireEvent, render } from '@testing-library/react';
+import { IPagedShortLinks } from '../../../service/ShortLink.service';
+
+describe('PreferenceToggle component', () => {
+  const TOGGLE_CLASS_NAME = '.toggle';
+  const TOGGLE_SELECTOR = TOGGLE_CLASS_NAME;
+  const TOGGLE_ACTIVE_SELECTOR = TOGGLE_CLASS_NAME + '> .active';
```

Comment:
let's move the setup code into the test themselves so that each test is independent from each other. It's okey to have repeatitation here.

Reply:
https://github.com/short-d/short#the-first-principal

Reply:
Done